### DISCUSSION TITLE
Activate LCD screen

### DISCUSSION THREAD

**Discussion by kobold77** [raspiblitz/raspiblitz#4042] [Q&A] [answered]
Hi together. 
I have set up a new node and accidentally installed the Minimal SD image. This one has the LCD and hdmi function disabled. 
I want to have the LCD function and have installed the fatpack. However, the display is still off and no display settings appear in the settings. 

How do I now activate the lcd screen?

**Comment by ghost** [comment, marked as answer]
Please type "lcd" after connecting via ssh and reboot.

**Comment by kobold77** [comment]
Hi, thanks for the quick reply. I did, the blitz has also activated the lcd. But after the reboot the lcd is still white.

**Comment by rootzoll** [comment]
If the LCD is still white check the following:
- are Pins connected correctly (all Pins on one side are used)
- is the LCD model correct (no HDMI, just Pins, it says XPT2046 controller on back)
